### PR TITLE
init container logging

### PR DESCRIPTION
**Pull request by dexhorthy**

### CODE REVIEW

**Comment by dexhorthy** [comment]
Will add a replicated-app.yaml

File: initcontainer-support-bundle-logs/manifests/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+Troubleshoting example: Multi-container Deployments
+========================
+
+
+This repo containers an example deployment and [troubleshot.sh](https://troubleshoot.sh) Collector. It demonstrates collecting logs from a multi-container deployment with two `initContainer` entries and two `container` entries.
```
Comment (aj-jester) [inline initcontainer-support-bundle-logs/manifests/README.md]: s/troubleshot/troubleshoot

Comment (aj-jester) [reply initcontainer-support-bundle-logs/manifests/README.md]: s/containers/contains

File: initcontainer-support-bundle-logs/manifests/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+after a few seconds, run the support bundle
+
+```
+kubectl support-bundle support-bundle.yaml
+```
+
+Untar the bundle
+
+```
+tar xvf support-bundle.tar.gz app-logs failing-app-logs
```
Comment (aj-jester) [inline initcontainer-support-bundle-logs/manifests/README.md]: `z` flag?

**Review by aj-jester — APPROVED** [review]
Minor comments but over all looks good 👍

File: initcontainer-support-bundle-logs/manifests/README.md (hunk 2 above)
Comment (dexhorthy) [reply initcontainer-support-bundle-logs/manifests/README.md]: ah yep, spoiled by BSD tar 😂

**Pull request merged**